docs/content/configs/recommended-custom.mdx

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
Can you also add the config to `docs/pages/configs/index.astro`?

Reply:
@azat-io Added in 8e64e71fdfdd8a244200dab9b80d894c33d3e658.